README.md

```diff
@@ -6,12 +6,25 @@ ROS Client Library for the Python language.
 
 Documentation can be built for `rclpy` using [Sphinx](http://www.sphinx-doc.org/en/master/), or accessed [online](http://docs.ros2.org/latest/api/rclpy/index.html)
 
+For building documentation, you need an installation of ROS 2.
+
 #### Install dependencies
 
     sudo apt install python3-sphinx python3-pip
     sudo -H pip3 install sphinx_autodoc_typehints
 
 #### Build
 
-    cd rclpy/docs
+Build code:
+
+    mkdir -p rclpy_ws/src
+    cd rclpy_ws/src
+    git clone https://github.com/ros2/rclpy.git
+    cd ..
+    colcon build --symlink-install
```

Comment:
The underlay needs to be sourced before.

Reply:
See 037825f